src/test/scala/no/gdl/bookapi/service/translation/TranslationServiceTest.scala

```diff
@@ -154,4 +155,124 @@ class TranslationServiceTest extends UnitSuite with TestEnvironment {
     val result = service.updateTranslationStatus("abc", LanguageTag("nob"), "123", TranslationStatus.TRANSLATED)
     result should be a 'Success
   }
+
+  test("that fetchTranslationsIfAllTranslated returns Success when not all translated") {
```

Comment:
This appears as a contradiction to me. `fetchTranslationsIfAllTranslated` sounds like it shouldn't return a `Success`?